uaclient/entitlements/base.py

```diff
@@ -273,7 +278,26 @@ def process_contract_deltas(
 
         return False
 
-    @abc.abstractmethod
     def operational_status(self) -> 'Tuple[str, str]':
         """Return whether entitlement is ACTIVE, INACTIVE or UNAVAILABLE"""
+        applicability, details = self.applicability_status()
```

Comment:
Always good to drop the duplicated code. thanks.

Reply:
:-)